### ISSUE TITLE
Add new functions in different .md files

### ISSUE THREAD

**Issue by Bhupesh-V** [Bhupesh-V/30-Seconds-Of-STL#31]
Add functions which belong to a particular header file in its corresponding _.md_ file.
For example 
**std::sort belongs to <algorithm>**, shift this to _algorithm.md_ while maintaining the [CONTIBUTING](https://github.com/Bhupesh-V/30-Seconds-Of-STL/blob/master/CONTRIBUTING.md) guidelines and formats.

**Comment by willdarnell** [comment]
are you saying you want the sort in sort.md moved to algorithm.md?

**Comment by dantedallag** [comment]
Is this issue solved? If not, I could try to do it.

**Comment by Bhupesh-V** [comment]
@dantedallag this issue always remains open to help contributors .

**Comment by HarshitNITT** [comment]
Can I contribute to this issue?

**Comment by Bhupesh-V** [comment]
> Can I contribute to this issue?

Yes you can, just see the unimplemented functions in the `todo.txt` files inside different directories

**Comment by saisriram2710** [comment]
Hi, I am new to open source in general, and I would like to contribute as a beginner. 
I have a few doubts:
1) How can I differentiate which functions are unimplemented in the `README.md` file?
2) I also want to make sure if the process of contributing that I understand is right or not: At the time of writing this, the pop_back() function is not yet implemented yet. So is this what I do - I write a sample code that implements this function (by following the coding conventions of course), and this code will reside in `snippet/vector`. And I also make a .md file in `vector` directory (outside snippet) which explains the code. Is there anything that I am missing? If yes, please tell me so that I begin.

**Comment by saisriram2710** [comment]
Hi, just an update: 
1. I made `pop_back.cpp` in `snippet/vector`. I also wrote the corresponding `.md` file in `vector` directory, in which I pasted the link to run the code on rextester. Is this enough? Is submitting a PR the job left to do or anything missing?

**Comment by Bhupesh-V** [comment]
@saisriram2710 you are right

**Comment by agrawalsajal02** [comment]
Can i contribute to this issue ?

**Comment by sakibmahmud** [comment]
Hi, I want to contribute to this issue. Where do i start?

**Comment by satcasm** [comment]
can i contribute to this issue??

**Comment by marcinky** [comment]
Hello! I would like to contribute to this issue. I have reviewed the CONTRIBUTING.md, looked at the Projects, and referenced the Kanban for each directory type. I would like to work on "pop_back" under Vector. If successful, this will be my very first Open Source contribution :) 
Thank you @Bhupesh-V for making this opportunity available. So far, the instructions have been very complete and helpful.

**Comment by Bhupesh-V** [comment]
> Hello! I would like to contribute to this issue. I have reviewed the CONTRIBUTING.md, looked at the Projects, and referenced the Kanban for each directory type. I would like to work on "pop_back" under Vector. If successful, this will be my very first Open Source contribution :)
> Thank you @Bhupesh-V for making this opportunity available. So far, the instructions have been very complete and helpful.

@marcinky looks like `pop_back` is already implemented and only `crend` is available to contribute.
Don't worry just head over to specific directories like [list](https://github.com/Bhupesh-V/30-seconds-of-cpp/tree/master/list) where pop_back has not been implemented yet (see :heavy_check_mark: and :x:  marks)
Go ahead and implement it and welcome to opensource :hugs:

**Comment by marcinky** [comment]
Ok will do @Bhupesh-V  - thank you! I'll take another look.

**Comment by dbejanishvili** [comment]
Should I create a new issue before starting to work on code or should I just go ahead and open a pull request?

**Comment by athulsasi** [comment]
Hi Bhupesh Varshney, 
I would like to contribute to this issue. 

I've locally created .md file with an example code for the new 'contains' in the _unordered_map_ for C++20.
I've followed your coding guideline and also tested my code. I can create a pull request for review.

Could you please allow me write access to create a PR? Thank you for your help in advance.

**Comment by akankshasonkar17** [comment]
Hi Bhupesh Varshney,
I just made my first PR for unordered_map 's at function . Please check it out . :)

https://github.com/Bhupesh-V/30-seconds-of-cpp/pull/579

**Comment by Bhupesh-V** [comment]
> Hi Bhupesh Varshney,
> I would like to contribute to this issue.
> 
> I've locally created .md file with an example code for the new 'contains' in the _unordered_map_ for C++20.
> I've followed your coding guideline and also tested my code. I can create a pull request for review.
> 
> Could you please allow me write access to create a PR? Thank you for your help in advance.

Hi @athulsasi you don't need any write access to the repo. 
Everything is listed in CONTRIBUTING guidelines on how & what to contribute.

**Comment by Aokison** [comment]
Hello @Bhupesh-V. I am completely new to open source and would like to contribute to this project, specifically the crend function in the set directory. I really appreciate the opportunity provided for newcomers like me.

**Comment by varunbgit** [comment]
@Bhupesh-V I have created a PR please review and suggest changes.
#649

**Comment by deepanshu2810** [comment]
@Bhupesh-V I would like to work on this one .

**Comment by Wadiya003** [comment]
Hi @Bhupesh-V  I would like to give it a try! :)

**Comment by Anshul-9923** [comment]
Hi @Bhupesh-V Can I contribute to this issue a function to unordered_map?

**Comment by Aanya9693** [comment]
@Bhupesh-V,Hey! I would like to contribute to this, please assign this to me.
